src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -203,10 +203,16 @@ FullyConnected_bf_tiled::GetAutoTuneParams(const fully_connected_params& params,
     if (params.is_shape_agnostic) {
         if (dtype == Datatype::F16) {
             // tune_params(tile_b, tile_ofm, tile_ifm, tile_k, dispatch_bsv, dispatch_fsv, exec_options)
-            selector.Case(tune_params(8,  std::min(max_tile_ofm, 2u), 1, 2, 1,  1, EXE_MODE_AGE_BASED));
+            if (params.engineInfo.supports_immad)
```

Comment:
1. Please add some comment which explains the motivation of different behavior.
2. Shouldn't it depend on EUs count rather then available ISA? What is more efficient on dGPU with 128 EUs?

Reply:
I have added some additional info
As for dGPU with 128 EUs - it also has some performance gain. That is why I would like to apply this tuning params for whole Gen12HP generation. Maybe other platforms will get some gain from this tuning as well, but I haven't checked it yet